CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Provide better error output on initial MQTT connection errors
 - Show total amount of messages in the topic overview title
+- Log verbose messages to stderr instead of stdout
+- Add an option to the `log` subcommand for outputting newline-delimited JSON logs
```

Comment:
I used the scope of the change as a prefix (and forgot on the line before, can you add Interactive: there while we are at it?).

`--json` seems like the more interesting change, so I would put it first. Guess I would have forgotten about the stdout to stderr change with this :innocent: 

Also used a different phrasing including the `--flags` so it's maybe a bit easier to recognize?

```suggestion
- Log: Provide machine-readable newline-delimited output with `--json`
- Log: Print `--verbose` to stderr instead of stdout
```